### PR TITLE
fix: deployment script updated

### PR DESCRIPTION
**Pull request by gcranju**
### Commit Message

```bash
fix: deployment script updated
```
see the [guidelines](https://github.com/icon-project/community/blob/main/guidelines/technical/software-development-guidelines.md#commit-messages) for commit messages.

### Changelog Entry

```bash
version: <log entry>
```

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have documented my code in accordance with the [documentation guidelines](https://github.com/icon-project/community/blob/main/guidelines/technical/software-development-guidelines.md#documentation)
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have run the unit tests
- [ ] I only have one commit (if not, squash them into one commit).
- [ ] I have a descriptive commit message that adheres to the [commit message guidelines](https://github.com/icon-project/community/blob/main/guidelines/technical/software-development-guidelines.md#commit-messages)

> Please review the [CONTRIBUTING.md](/CONTRIBUTING.md) file for detailed contributing guidelines.

### CODE REVIEW

File: contracts/evm/xcall/script/CallService.s.sol [hunk 1]
```diff
@@ -8,8 +8,14 @@ contract CallServiceScript is Script {
...
 
     function run() public {
-        vm.broadcast();
-        new CallService();
+        uint256 deployerPrivateKey = vm.envUint("PRIVATE_KEY");
+        vm.startBroadcast(deployerPrivateKey);
+
+        CallService xcall = new CallService();
+
+        xcall.initialize("bsc");
```
Comment (AntonAndell) [inline contracts/evm/xcall/script/CallService.s.sol:16]: @sdpisreddevil This should be the nid right? Do we want it hardcoded like this?

**Review by AntonAndell — CHANGES_REQUESTED** [review]

File: contracts/evm/xcall/script/CallService.s.sol (hunk 1 above)
Comment (sdpisreddevil) [reply contracts/evm/xcall/script/CallService.s.sol:16]: No, this must be the NID. Should be taken from env too

**Comment by sdpisreddevil** [comment]
Contract verification parts can be added later, looks good for right now.

**Review by AntonAndell — APPROVED** [review]
LGTM

**Pull request merged**